datalake/src/main/java/com/sequenceiq/datalake/flow/dr/validation/handler/DatalakeBackupValidationWaitHandler.java

```diff
@@ -0,0 +1,83 @@
+package com.sequenceiq.datalake.flow.dr.validation.handler;
+
+import static com.sequenceiq.datalake.flow.dr.validation.DatalakeBackupValidationEvent.DATALAKE_BACKUP_VALIDATION_SUCCESS_EVENT;
+
+import java.util.concurrent.TimeUnit;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import com.dyngr.exception.PollerException;
+import com.dyngr.exception.PollerStoppedException;
+import com.dyngr.exception.UserBreakException;
+import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.cloudbreak.exception.CloudbreakApiException;
+import com.sequenceiq.datalake.flow.SdxEvent;
+import com.sequenceiq.datalake.flow.dr.validation.event.DatalakeBackupValidationFailedEvent;
+import com.sequenceiq.datalake.flow.dr.validation.event.DatalakeBackupValidationWaitRequest;
+import com.sequenceiq.datalake.service.sdx.PollingConfig;
+import com.sequenceiq.datalake.service.sdx.dr.SdxBackupRestoreService;
+import com.sequenceiq.flow.event.EventSelectorUtil;
+import com.sequenceiq.flow.reactor.api.handler.ExceptionCatcherEventHandler;
+import com.sequenceiq.flow.reactor.api.handler.HandlerEvent;
+
+import reactor.bus.Event;
+
+@Component
+public class DatalakeBackupValidationWaitHandler extends ExceptionCatcherEventHandler<DatalakeBackupValidationWaitRequest> {
```

Comment:
Since we're doing validation only, the backup should be complete with the original response. Right now we don't need this waiter.

Reply:
Verifying with Kalyan on what to do for this.

Reply:
Ah you're right in most situations in might not actually be syncronous.